kuksa-client/README.md

```diff
@@ -33,7 +33,7 @@ If you wish to connect to a gRPC server e.g. `kuksa-databroker`, you should inst
 ```console
 $ kuksa-client --ip 127.0.0.1 --port 55555 --protocol grpc --insecure
 ```
-Note: `--insecure` is required because `kuksa-databroker` does not yet support encryption or authentication.
+Note: `--insecure` is required because `kuksa-databroker` does not yet support TLS encryption or authentication.
```

Comment:
this isn't true anymore or am I wrong. Databroker supports TLS but the client not at this time?

Reply:
Correct, or at least not with the current default certificates. I am touching this file in https://github.com/eclipse/kuksa.val/pull/580/files#diff-9e56616746625998b6fde6fba9bd42bf7385758ccb26a51ef83462310f06123c as well, but it could better be rephrased here